Software/2.Unity/Assets/testLoadResources.cs

```diff
@@ -191,8 +229,8 @@ private void readAllFileTxt()
             //Debug.Log(data[i].Substring(data[i].LastIndexOf(" ") + 1));
         }
     }
-    //x so tang de
-    //y so tang toa nha
+    //x number footprint
+    //y number floor
     private void Calculator(int x, int y)
     {
         kcx = 139 / (x + (7f / 12f) * y);
```

Comment:
what does this calculator use for?

Reply:
As you can see, the two input parameters here are the footprint and the number of floors of the building. In this case, there are two variables x and y.
Compared to the standard building as the base (the building that Mr. Hai Hoang said), it has a height of 139 in unity. It will rely on 2 input parameters x and y to calculate the distance between floors and the distance between Footprint layers are reasonable, based on the original house